src/js/directives/autoHideShow.js

```diff
@@ -0,0 +1,147 @@
+/** @module AutoHideShowControls
+* Directive used to autohide the UI. Can be instantiated by setting data-mstr-directive="autoHideShow" on an element.
+*/
+
+let TIME_TILL_FADE = 3000;
```

Comment:
This does mean that the time to fade can't be configured separately between instances. Not sure that's an issue, but I figured it was worth noting.